### PR TITLE
Improve variable name

### PR DESCRIPTION
**Pull request by fabienrohrer**
Following a comment of @omichel

### CODE REVIEW

**Comment by fabienrohrer** [comment]
👍

File: projects/robots/telerob/telemax_pro/controllers/telemax_demo/telemax_demo.c [hunk 1]
```diff
@@ -24,10 +24,10 @@
 
 #define TIME_STEP 32
 
-void noop(int n_steps) {
-  // Wait without applying commands during 'n_steps' steps.
+void noop(int duration) {
+  // Wait without applying any command during 'duration' simulation steps.
```
Comment (omichel) [inline projects/robots/telerob/telemax_pro/controllers/telemax_demo/telemax_demo.c]:
```suggestion
  // Wait without applying any command during 'duration' milliseconds.
```

Comment (fabienrohrer) [reply projects/robots/telerob/telemax_pro/controllers/telemax_demo/telemax_demo.c]: Your suggestion is wrong, the unit of the argument is a number of simulation steps, not milliseconds. Please check the function content.

Comment (fabienrohrer) [reply projects/robots/telerob/telemax_pro/controllers/telemax_demo/telemax_demo.c]: Therefore `n_steps` is better than `duration` IMO

Comment (omichel) [reply projects/robots/telerob/telemax_pro/controllers/telemax_demo/telemax_demo.c]: I don't think so: `counter` is accumulating `TIME_STEP` which is expressed in milliseconds and then compared to `duration` which should therefore be milliseconds as well.

Comment (fabienrohrer) [reply projects/robots/telerob/telemax_pro/controllers/telemax_demo/telemax_demo.c]: Oops, you're right. Sorry for this.

**Comment by fabienrohrer** [comment]
👍

**Pull request merged**